docs/content.zh/docs/dev/python/faq.md

```diff
@@ -32,7 +32,7 @@ under the License.
 
 ## 准备Python虚拟环境
 
-您可以下载[便捷脚本]({% link downloads/setup-pyflink-virtual-env.sh %})，以准备可在Mac OS和大多数Linux发行版上使用的Python虚拟环境包(virtual env zip)。
+您可以下载[便捷脚本](https://nightlies.apache.org/flink/flink-docs-release-1.12/downloads/setup-pyflink-virtual-env.sh)，以准备可在Mac OS和大多数Linux发行版上使用的Python虚拟环境包(virtual env zip)。
```

Comment:
It's a little tricky to refer to link in previous releases.

Reply:
I agree. The reason for such use is that I checked our current doc based on hugo, and It seems that there is no out-of-box place to save this script. 
Maybe we can refer to the native k8s' arrangement of [podTemplate](https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/resource-providers/native_kubernetes/#example-of-pod-template) to directly put the script in the content, which is more straightforward. What's your opinion?